### ISSUE TITLE
probably won't run on all distros.

### ISSUE THREAD

**Issue by gdoenlen** [robraddi/SymPyVolume#2]
most distros have `/usr/bin/env python` pointing to python2 and it looks like you're using a python3 class definition.

**Comment by robraddi** [comment]
Yes, especially so for the way the [file name is created](https://github.com/robraddi/SymPyVolume/blob/master/plot.py#L188).
I had to change the multiplication symbols (and others) that gave me issues when working the name. Probably don't need to name the file after the function, but that was my first instinct a few years back. (very old project lol)

**Comment by gdoenlen** [comment]
It probably doesn't matter in the long run. Since py2 should be deprecated next year, but it's something to keep in mind.
